jaxbind/contrib/jaxducc0.py

```diff
@@ -30,6 +50,20 @@ def _realtype(dtype):
     return _c2rdict[np.dtype(dtype)]
 
 
+# %% [markdown]
+
+# ## Binding the DUCC hartly transform to JAX
+
+# In the following we provide a Python `_fht` calling C++ hartly transformation
+# of DUCC ` ducc0.fft.genuine_fht`. The C++ function natively supports
+# batching along axis. To make use of that the the python function also
+# translates potential batching axis of JAX to axis of DUCC. Detailed
+# explanations on custom batching along axis can be found in the
```

Comment:
axis should probably be either "an axis" or "multiple axes"

Reply:
Thanks